### DISCUSSION TITLE
Is it possible for Kobo Aura H2O to boot directly to Koreader?

### DISCUSSION THREAD

**Discussion by kartikynwa** [koreader/koreader#12137] [Q&A]
I think it's outside the scope of Koreader but input is appreciated.

Whenever I boot up the ereader, I ALWAYS select Koreader and start it from Nickel. Would be nice to be able to skip Nickel or automate the process. It is not a big deal at all. Just asking out of curiosity.

Thanks.

**Comment by jonnyl2** [comment]
Something similar was asked [here](https://github.com/koreader/koreader/discussions/12006#discussion-6801503).

Whether launching KOReader can be automated on Nickel on start-up I don't know, but always keep in mind that there needs to be an option to cancel the automatic launch, in case there is ever a problem with your KOReader installation that you could still easily fix by connecting your device to the computer via Nickel.

**Reply by Frenzie** [reply]
The crude method is to edit /etc/init.d/rcS according to your liking, but it's hardly worth the risk of having to perform a full factory reset through convoluted methods if you mess up.
